translations/de/task_short_description.html

```diff
@@ -1 +1,2 @@
-Division, great for numbers, food, labor, and buried treasure. Not so much for politics.
\ No newline at end of file
+Teilen, großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. Nicht so sehr in der
```

Comment:
Da weder Zahlen, Essen, Arbeit noch Schätze etwas davon haben geteilt zu werden funktioniert **_für_** hier nicht wirklich.

**_Mit_** geht auch nicht nicht wirklich... Denn das Problem liegt ja nicht bei der Politik, sondern bei den Politikern.

Wie wärs hiermit:  
_Teilen ist großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. In der Politik ist es aber unbeliebt._